strings/src/main/java/com/revenuecat/purchases/strings/IdentityStrings.kt

```diff
@@ -10,4 +10,6 @@ object IdentityStrings {
     const val SETTING_NEW_ANON_ID = "Setting new anonymous App User ID - %s"
     const val LOG_OUT_CALLED_ON_ANONYMOUS_USER = "Called logOut but the current user is anonymous"
     const val LOG_OUT_SUCCESSFUL = "Logged out successfully"
+    const val INVALIDATING_CACHED_CUSTOMER_INFO = "Detected cached unverified CustomerInfo but verification " +
```

Comment:
I think this would read a bit better like
```suggestion
    const val INVALIDATING_CACHED_CUSTOMER_INFO = "Detected unverified cached CustomerInfo but verification " +
```